requirements.txt

```diff
@@ -1,30 +1,36 @@
-alembic==1.8.1
-alembic-utils==0.7.8
+alembic==1.9.4
```

Comment:
This comment applies to requirements.txt, requirements-app.txt, and requirements_for_test.txt. 

I don't understand why so many dependencies are changed for Datadog APM. You had mentioned that you had merged master into this branch, but none of these changes are on master (which is why they are showing as changed in this PR). 

We would need extensive testing to ensure compatibility here and the Flask upgrade might be taking care of most of these. 

Can these files only be changed for what DD APM actually needs?